src/Resources/views/storefront/layout/meta.html.twig

```diff
@@ -258,6 +247,20 @@
                     });
                   }
                 }
+                if ('phoneServices' === markerValue) {
+                    var countryCodeSelector = MarkerElement.getAttribute('data-country-code-selector');
+                    var phoneSelector = MarkerElement.getAttribute('data-phone-selector');
+                    var numberType = MarkerElement.getAttribute('data-number-type') ?? window.EnderecoIntegrator.config.defaultPhoneType;
+                    window.EnderecoIntegrator.initPhoneServices(
+                        {
+                            phone: `[data-endereco-${markerValue}-form-id="${formId}"] [name="${phoneSelector}"]`,
+                            countryCode: `[data-endereco-${markerValue}-form-id="${formId}"] [name="${countryCodeSelector}"]`
+                        }, {
+                            name: `phs_${prefix}_${formId}`,
+                            numberType,
```

Comment:
This looks broken. I would have expected `numberType: numberType`. Is this typo?

Reply:
It was intended, something like this is called "Object Literal Property Value Shorthand". It means when I got situation like this when I want to assign property with same name as variable, I don't need to use it as `numerType: numberType,` but I can shorthand it to just `numberType`. Here is more detailed explanation: https://attacomsian.com/blog/javascript-object-property-shorthand